src/sentry/api/endpoints/organization_spans_fields.py

```diff
@@ -181,6 +183,67 @@ def get(self, request: Request, organization, key: str) -> Response:
 
         max_span_tag_values = options.get("performance.spans-tags-values.max")
 
+        serializer = OrganizationSpansFieldsEndpointSerializer(data=request.GET)
+        if not serializer.is_valid():
+            return Response(serializer.errors, status=400)
+        serialized = serializer.validated_data
+
+        if serialized["dataset"] == "spans" and features.has(
+            "organizations:visibility-explore-dataset", organization, actor=request.user
+        ):
+            start_timestamp = Timestamp()
+            start_timestamp.FromDatetime(
+                snuba_params.start_date.replace(hour=0, minute=0, second=0, microsecond=0)
+            )
+
+            end_timestamp = Timestamp()
+            end_timestamp.FromDatetime(
+                snuba_params.end_date.replace(hour=0, minute=0, second=0, microsecond=0)
+                + timedelta(days=1)
+            )
+
+            rpc_request = AttributeValuesRequest(
+                meta=RequestMeta(
+                    organization_id=organization.id,
+                    cogs_category="performance",
+                    referrer=Referrer.API_SPANS_TAG_KEYS.value,
+                    project_ids=snuba_params.project_ids,
+                    start_timestamp=start_timestamp,
+                    end_timestamp=end_timestamp,
+                    trace_item_name=TraceItemName.TRACE_ITEM_NAME_EAP_SPANS,
+                ),
+                name=key,
+                value_substring_match=request.GET.get("query", ""),
+                limit=max_span_tag_values,
+                offset=0,
+            )
+            rpc_response = snuba.rpc(rpc_request, AttributeValuesResponse)
+
+            paginator = ChainPaginator(
+                [
+                    [
+                        TagValue(
+                            key=key,
+                            value=tag_value,
+                            times_seen=None,
+                            first_seen=None,
+                            last_seen=None,
```

Comment:
Are these all None cause its not something we can query yet? if so could we add a TODO comment